app/questionnaire/rules.py

```diff
@@ -281,3 +290,30 @@ def get_list_count(list_store, list_name):
 
 def is_goto_rule(rule):
     return any(key in rule.get('goto', {}) for key in ('when', 'block', 'group'))
+
+
+def get_default_answer(answer_id, schema):
+    try:
+        block = schema.get_block_for_answer_id(answer_id)
+    except TypeError:
+        raise NoDefaultAnswerError
+
+    question = None
+
+    try:
+        question = block['question']
+    except KeyError:
+        variants = block['question_variants']
+        question = variants[0]['question']
+
+    if not question:
+        raise NoDefaultAnswerError
+
+    answers = [answer for answer in question['answers'] if answer['id'] == answer_id]
+
+    try:
+        answer = Answer(answers[0]['id'], answers[0]['default'])
+    except (KeyError, TypeError):
+        raise NoDefaultAnswerError
```

Comment:
I don't think we should be using the `NoDefaultAnswerError` exception throughout this method. It's actually more likely for a default answer to not exist. It would be better to just return `None`.